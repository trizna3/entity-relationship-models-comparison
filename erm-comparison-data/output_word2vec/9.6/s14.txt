account-accounts
real estate insurance-real estate insurance
real estate types-EMPTY
contract-contracts
persons;addresses-address
life insurance-life insurance
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estate insurance
CONTRACT_11_ASSOCIATION:E-persons-addresses
