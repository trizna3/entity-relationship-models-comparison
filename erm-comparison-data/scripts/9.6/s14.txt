entity set accounts {
	"login name"
	"password"
	"name"
	"surname"
};

entity set contracts {
	"date from"
	"date to"
	"fee"
	"current name"
	"current surname"
};

entity set "life insurance" {
	"death amount"
	"injury amount"
	"work incapacity amount"
	"current insured name"
	"current insured surname"
};

entity set "real estate insurance" {
	"crash amount"
	"nature amount"
	"address"
	"type"
	"living area size"
	"basement size"
};

association "signed contract" between 
{
	accounts ONE 
	contracts MANY
}{};

association "for whom" between 
{
	accounts ONE 
	"life insurance" MANY
}{};

generalization from "real estate insurance" to contracts;

generalization from "life insurance" to contracts;
