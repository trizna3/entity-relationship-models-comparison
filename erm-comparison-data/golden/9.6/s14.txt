account-accounts
contract-contracts
life insurance-life insurance
real estate insurance-real estate insurance
persons-EMPTY
addresses-address
real estate types-type
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-type-real estates insurance
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estates insurance